### PR TITLE
clone on entrypoint so this work doesn't have to go in behaviors

### PR DESCRIPTION
**Pull request by awsmsrc**
will soon be done with volumes though

### CODE REVIEW

File: clone.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash 
+
+echo "cloning git repo"
+git clone https://${GITHUB_TOKEN}@github.com/${GITHUB_USER}/${GITHUB_REPO}.git target
```
Comment (tommoor) [inline clone.sh]: should we set git author and checkout correct base branch here too?

Comment (awsmsrc) [inline clone.sh]: git author is already set in the Dockerfile,

We should check out the correct branch yes

Comment (awsmsrc) [inline clone.sh]: @tommoor Ive made that change

**Comment by tommoor** [comment]
lgtm, related - what do you think about naming this 'base-image' 'ruby-image' etc? repo names are a little too generic for my liking

**Pull request merged**